src/dotnet-svcutil/lib/tests/Baselines/ParamsFiles_SDK_TFM/_edb/_edb.params.json

```diff
@@ -1,18 +1,18 @@
 {
-  "providerId": "TFnet5_0",
+  "providerId": "_edb",
```

Comment:
why not use 6.0 to replace 5.0?  Is _edb defined somewhere, I might have missed it>